packages/component/src/Utils/activityAltText.ts

```diff
@@ -19,23 +20,34 @@ function allTextContents(element: Node): string[] {
   return results;
 }
 
-/** Returns the alt text for an activity. */
-export default function activityAltText(activity: any): false | string {
+/** Returns the alt text for a message activity. */
+export default function activityAltText(
+  activity: any,
+  renderMarkdownAsHTML: (markdown: string) => string
```

Comment:
Why is the hook being passed in, instead of called in this file?

Reply:
I saw later you removed the hook, could you describe?

Reply:
This is not React component, thus, we cannot call `useRenderMarkdownAsHTML`. We relies on our caller to pass it in.